Friendly-plans/app/src/androidTest/java/pg/autyzm/friendly_plans/TaskListActivityTest.java

```diff
@@ -0,0 +1,107 @@
+package pg.autyzm.friendly_plans;
+
+import android.content.Intent;
+import android.support.test.espresso.Espresso;
+import android.support.test.espresso.contrib.RecyclerViewActions;
+import android.support.test.rule.ActivityTestRule;
+import android.support.test.runner.AndroidJUnit4;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.ClassRule;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import database.entities.TaskTemplate;
+import database.repository.TaskTemplateRepository;
+
+import static android.support.test.espresso.Espresso.onView;
+import static android.support.test.espresso.action.ViewActions.click;
+import static android.support.test.espresso.assertion.ViewAssertions.matches;
+import static android.support.test.espresso.contrib.RecyclerViewActions.scrollToPosition;
+import static android.support.test.espresso.matcher.ViewMatchers.assertThat;
+import static android.support.test.espresso.matcher.ViewMatchers.hasDescendant;
+import static android.support.test.espresso.matcher.ViewMatchers.withId;
+import static android.support.test.espresso.matcher.ViewMatchers.withText;
+import static matcher.RecyclerViewMatcher.withRecyclerView;
+import static org.hamcrest.CoreMatchers.is;
+
+/**
+ * Created by Mateusz on 2017-03-13.
+ */
+@RunWith(AndroidJUnit4.class)
+public class TaskListActivityTest {
+
+    @ClassRule
+    public static DaoSessionResource daoSessionResource = new DaoSessionResource();
+
+    @Rule
+    public ActivityTestRule<TaskListActivity> activityRule = new ActivityTestRule<>(
+            TaskListActivity.class, true, true);
+
```

Comment:
I think we should left only one empty line between variables, methods and so on - please remove an additional one